test/test_storage.py

```diff
@@ -86,6 +85,87 @@ async def test_storage_message_duration_too_long(self, fetch_storage_by_target_i
+        message.guild.get_member = Mock(return_value=drone_member)
+
+        fixed_now = datetime.now()
+        mocked_datetime.now.return_value = fixed_now
+
+        # run & assert
+        self.assertFalse(await storage.store_drone(message))
+
+        message.guild.get_member.assert_called_once_with('3287snowflake')
+        drone_member.remove_roles.assert_called_once_with(drone_role, development_role)
+        drone_member.add_roles.assert_called_once_with(stored_role)
+        self.assertEqual(insert_storage.call_args.args[0].stored_by, "3287")
+        self.assertEqual(insert_storage.call_args.args[0].target_id, "3287")
+        self.assertEqual(insert_storage.call_args.args[0].purpose, "recharge")
+        self.assertEqual(insert_storage.call_args.args[0].roles, f"{roles.DRONE}|{roles.DEVELOPMENT}")
+        self.assertEqual(insert_storage.call_args.args[0].release_time, str(fixed_now + timedelta(hours=8)))
+        storage_chambers.send.assert_called_once_with("Greetings <3287mention>. You have been stored away in the Hive Storage Chambers by yourself for 8 hours and for the following reason: recharge")
+
+    @patch("ai.storage.datetime")
+    @patch("ai.storage.insert_storage")
+    @patch("ai.storage.fetch_drone_with_drone_id", return_value=Drone('3287snowflake', '3287', False, False, '', datetime.now()))
+    @patch("ai.storage.fetch_storage_by_target_id", return_value=None)
+    async def test_store_drone_hive_mxtress(self, fetch_storage_by_target_id, fetch_drone_with_drone_id, insert_storage, mocked_datetime):
+        # setup
+        message = AsyncMock()
+        message.channel.name = channels.STORAGE_FACILITY
+        message.content = "0006 :: 3287 :: 8 :: recharge"
+        message.author.roles = [drone_role]
+        message.guild.roles = [hive_mxtress_role, drone_role, development_role, stored_role]
+        message.guild.channels = [storage_chambers]
+
+        drone_member = AsyncMock()
+        drone_member.roles = [drone_role, development_role]
+        drone_member.mention = "<3287mention>"
+        message.guild.get_member = Mock(return_value=drone_member)
+
+        fixed_now = datetime.now()
+        mocked_datetime.now.return_value = fixed_now
+
+        # run & assert
+        self.assertFalse(await storage.store_drone(message))
+
+        message.guild.get_member.assert_called_once_with('3287snowflake')
+        drone_member.remove_roles.assert_called_once_with(drone_role, development_role)
+        drone_member.add_roles.assert_called_once_with(stored_role)
+        self.assertEqual(insert_storage.call_args.args[0].stored_by, "0006")
+        self.assertEqual(insert_storage.call_args.args[0].target_id, "3287")
+        self.assertEqual(insert_storage.call_args.args[0].purpose, "recharge")
+        self.assertEqual(insert_storage.call_args.args[0].roles, f"{roles.DRONE}|{roles.DEVELOPMENT}")
+        self.assertEqual(insert_storage.call_args.args[0].release_time, str(fixed_now + timedelta(hours=8)))
```

Comment:
it would be easier to read and maintain to extract `insert_storage.call_args.args[0]` into a variable